linux/shim/init.go

```diff
@@ -229,6 +229,7 @@ func (p *initProcess) SetExited(status int) {
 	p.mu.Lock()
 	p.status = status
 	p.exited = time.Now()
+	p.platform.shutdownConsole(context.Background(), p.console)
```

Comment:
close the console here looks right. I think i added this in the wrong place. I think we should remove this line in `*initProcess.Delete`